modules/040-control-plane-manager/hooks/discover_modules.go

```diff
@@ -88,6 +91,9 @@ func handleAuthDiscoveryModules(input *go_hook.HookInput) error {
 		userAuthnOIDCIssuerURLPath     = "controlPlaneManager.apiserver.authn.oidcIssuerURL"
 		userAuthnOIDCIssuerAddressPath = "controlPlaneManager.apiserver.authn.oidcIssuerAddress"
 		userAuthnOIDCIssuerCAPath      = "controlPlaneManager.apiserver.authn.oidcCA"
+
+		runtimeAuditWebhookURLPath = "controlPlaneManager.internal.audit.webhookURL"
+		runtimeAuditWebhookCAPath  = "controlPlaneManager.internal.audit.webhookCA"
```

Comment:
Should we use similar paths ? 
controlPlaneManager.internal.authn ?
or
controlPlaneManager.apiserver.audit ?

Reply:
.apiserver for values that came from the module config.
.internal for values discovered by a hook. 

I didn't add the ability to configure audit webhook options via module config. Thus, these values are internal.